webpack.config.js

```diff
@@ -1,14 +1,40 @@
 const path = require("path");
 const ESLintPlugin = require("eslint-webpack-plugin");
 const CopyPlugin = require("copy-webpack-plugin");
+const UnminifiedWebpackPlugin = require("unminified-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 module.exports = {
-  entry: path.resolve(__dirname, "src/index.js"),
+  entry: {
+    "formio-qld.min": {
+      import: path.resolve(__dirname, "src/index.js"),
+      library: {
+        name: "FormioQld",
+        type: "umd",
+      },
+    },
+    "formio-loader.min": {
+      import: path.resolve(
+        __dirname,
+        "src/matrixHelpers/formioLoader/index.js"
+      ),
+      library: {
+        name: "FormioLoader",
+        type: "umd",
+      },
+    },
+    "formio-script.dev.min": path.resolve(
+      __dirname,
+      "src/matrixHelpers/formioScript/index.dev.js"
+    ),
+    "formio-script.prod.min": path.resolve(
```

Comment:
we also have 
staging: beta-static.qgov.net.au
test: dev-static.qgov.net.au

can you also add these entry points.

Reply:
i think you've misunderstood, the entry point here is for compiling and bundling JS files